Software-Engineering/L5-Principal-Software-Engineer.md

```diff
@@ -1,97 +1,124 @@
 # L5: Principal Software Engineer
 
-> _I am an influential Engineering leader who identifies problems and opportunities relevant to Octopus' strategy, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus._
+> _I am an influential Engineering leader who identifies problems and opportunities relevant to Octopus product and technical strategies, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus._
 
-- **Planning horizon**: 1+ year
-- **Impact radius**: Group+ (15-50). Note: This role is evolving as the company grows. PSE is right at the top of our IC track. For example, the impact radius expectations of an L5 will likely grow as Octopus grows larger over the next couple years, from 2-4 teams in 2021. We expect the role to stabilise in 2022 once the company has grown to the point of needing an L6 IC engineering role.
-- **Evaluation**: Director or VP
-- **Responsibility and direction needed**: Drives technical strategy in alignment with product strategy and drives high-impact improvements across teams.
+- **Thinking horizon**: 1 year
+- **Impact radius**: Group
+- **Evaluation**: Director
+- **Responsibility and direction needed**: I drive technical strategy in alignment with product strategy for my group. Point me in the right direction with helpful constraints and good outcomes happen with minimal oversight.
+- **Links**: [2023/24 Octopus Technical Strategy](https://docs.google.com/document/d/1onaZ7qRWc7BSLWyGw8duIIlOCd2k812zMIXXtlBfXL8/edit#heading=h.a65d8nqv4wgr) | [Target Architecture](https://github.com/OctopusDeploy/OctopusDeploy/tree/main/docs/target-architecture)
```

Comment:
On first read, it's unclear why these links are here. The other bullet points refer to technical strategy "for my group", but these links refer to R&D-wide technical strategy. This might muddy the waters, unless we clarify explicitly.

Reply:
Why I added something: We refer to "Octopus technical strategy" and "Octopus target architecture" throughout this role description, but don't define those terms. This was an attempt to call these out as important terms that aid understanding of the role.

Instead of "links", what about "definitions" or "important terms"? Suggestions welcome!

Reply:
Ah, right, thank you. 

I was confused initially because PEs own _group-level_ technical strategy, so it felt surprising to see _R&D-level_ technical strategy called out right alongside the impact radius here.

In that case, suggestion:

```suggestion
- **Glossary (examples from 2023/24)**: ["Octopus Technical Strategy"](https://docs.google.com/document/d/1onaZ7qRWc7BSLWyGw8duIIlOCd2k812zMIXXtlBfXL8/edit#heading=h.a65d8nqv4wgr) | ["Target Architecture"](https://github.com/OctopusDeploy/OctopusDeploy/tree/main/docs/target-architecture)
```